build-system/amp4test.js

```diff
@@ -17,15 +17,33 @@
 
 const app = module.exports = require('express').Router();
 
-app.get('/compose-doc', function(req, res) {
+app.use('/compose-doc', function(req, res) {
+  res.setHeader('X-XSS-Protection', '0');
+
+  const meta = req.query.meta;
+  const experiments = req.query.experiments.split(' ');
+  let metaTag;
+  if (meta != '' || experiments != ['']) {
+    metaTag = '<meta ' + meta + ' content="' +
+          experiments.map(function(experiment) {
+            return experiment;
+          }).join(',') + '">';
+  }
+
   res.send(`
 <!doctype html>
 <html ⚡>
 <head>
   <meta charset="utf-8">
   <link rel="canonical" href="http://nonblocking.io/" >
-  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
-  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">`
+      + metaTag +
+      `<script>
+        window.AMP_CONFIG = window.AMP_CONFIG || {
+            ['allow-doc-opt-in']: experiments.join(',')
```

Comment:
from the test file (e.g: experiments: ['user-error', 'sidebar']), to pass the experiment id to server